mmv1/third_party/terraform/tests/resource_cloud_run_domain_mapping_test.go

```diff
@@ -34,15 +33,11 @@ func TestAccCloudRunDomainMapping_foregroundDeletion(t *testing.T) {
 				ImportStateVerifyIgnore: []string{"name", "location", "status", "metadata.0.resource_version"},
 			},
 			{
-				Config:      testAccCloudRunDomainMapping_cloudRunDomainMappingUpdated2(context),
```

Comment:
We want to keep the config (we're testing that updating works!) but drop the ExpectError here btw

Reply:
ohh ya, that makes more sense